docs/src/glasscat.md

```diff
@@ -1,12 +1,38 @@
 # GlassCat
 
-Julia module for importing and using AGF glass specifications.
+This submodule is used to download, parse, install and manage AGF glass specifications for use in OpticSim.
 
-The entire AGF glass catalog is specified in `AGFGlassCat.jl`. This Julia source file is generated automatically when `] build OpticSim` is called. The build script downloads AGF files to `deps/downloads/glasscat/` and then uses these to generate corresponding Julia source files at `src/GlassCat/data/`. These steps are run automatically on setup when the package is first installed using `] add OpticSim`, creating a sufficient working environment for our example/test code.
+The central configuration file for GlassCat is located at `src/GlassCat/data/sources.txt`, which ships with the
+following default entries.
 
-Adding new AGF sources is done by editing `deps/sources.txt`. Minimally, you must provide a name (e.g. SCHOTT) and sha256sum for the AGF file, which can then be placed manually into `deps/downloads/glasscat/[NAME].agf`. Instead of manually sourcing the AGF file, you can also provide a download link for the build script. `deps/sources.txt` already contains examples of all possible use cases. After updating the file, execute `] build OpticSim` to rebuild `AGFGlassCat.jl`.
+```
+NIKON a49714470fa875ad4fd8d11cbc0edbf1adfe877f42926d7612c1649dd9441e75 https://www.nikon.com/products/components/assets/pdf/nikon_zemax_data.zip
+OHARA 0c9021bf11b8d4e660012191818685ad3110d4f9490699cabdc89aae1fd26d2e https://www.oharacorp.com/xls/OHARA_201130_CATALOG.zip
+HOYA b02c203e5a5b7a8918cc786badf0a9db1fe2572372c1c163dc306b0a8a908854 http://www.hoya-opticalworld.com/common/agf/HOYA20210105.agf
+SCHOTT e9aabbb8ebff116ba0c106a71afd86e72f2a397ac9bc447469129e325e795f4e https://www.schott.com/d/advanced_optics/6959f9a4-0e4f-4ef2-a302-2347468a82f5/1.31/schott-optical-glass-overview-zemax-format.zip
+Sumita c1093e42a1d08acbe30698aba730161e3b43f8f0d50533f65de8b6b11100fdc8 https://www.sumita-opt.co.jp/en/wp/wp-content/themes/sumita-opt-en/download-files.php files%5B%5D=new_sumita.agf&category=data
+```
+
+Each line corresponds to one AGF source, which is described by 2 to 4 space-delimited columns. The first column provides
+the installed module name for the catalog, e.g. `GlassCat.NIKON`. The second column is the expected SHA256 checksum for
+the AGF file.
+
+The final two columns are optional, specifying download instructions for acquiring the zipped AGF files
+automatically from the web. The fourth column allows us to use POST requests to acquire files from interactive sites.
+
+When `] build OpticSim` is run, the sources are verified and parsed into corresponding Julia files. These are then
+included in OpticSim via `AGFGlassCat.jl`. These steps are run automatically when the package is first installed using
+`] add OpticSim`, creating a sufficient working environment for our examples and tests.
+
+## Adding glass catalogs
```

Comment:
@alfredclwong does this just add a filename to the text file or does it also trigger a build of the AGFGlassCat.jl file?